From: Richard Acayan <mailingradian@gmail.com>
Date: Mon, 17 Oct 2022 18:52:23 -0400
Subject: [PATCH] arm64: dts: qcom: sdm670-google: enable venus video codec

Venus is present in the Google Pixel 3a. Enable the device tree node so
it can be accessed by software.

Signed-off-by: Richard Acayan <mailingradian@gmail.com>
---
 arch/arm64/boot/dts/qcom/sdm670-google-common.dtsi | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/arch/arm64/boot/dts/qcom/sdm670-google-common.dtsi b/arch/arm64/boot/dts/qcom/sdm670-google-common.dtsi
index 6c2762e..491b1fa 100644
--- a/arch/arm64/boot/dts/qcom/sdm670-google-common.dtsi
+++ b/arch/arm64/boot/dts/qcom/sdm670-google-common.dtsi
@@ -782,3 +782,8 @@ &usb_1_dwc3 {
 	phy-names = "usb2-phy";
 	maximum-speed = "high-speed";
 };
+
+&venus {
+	firmware-name = "qcom/sdm670/sargo/venus.mbn";
+	status = "okay";
+};
